recipes/orfm/build.sh

```diff
@@ -1,24 +1,21 @@
 #!/bin/bash
 
-# fix automake
-sed -i.bak '1 s|^.*$|#!/usr/bin/env perl|g' $PREFIX/bin/aclocal
-sed -i.bak '1 s|^.*$|#!/usr/bin/env perl|g' $PREFIX/bin/automake
-
-# fix autoconf
-sed -i.bak '1 s|^.*$|#!/usr/bin/env perl|g' $PREFIX/bin/autom4te
-sed -i.bak '1 s|^.*$|#!/usr/bin/env perl|g' $PREFIX/bin/autoheader
-sed -i.bak '1 s|^.*$|#!/usr/bin/env perl|g' $PREFIX/bin/autoreconf
-sed -i.bak '1 s|^.*$|#!/usr/bin/env perl|g' $PREFIX/bin/ifnames
-sed -i.bak '1 s|^.*$|#!/usr/bin/env perl|g' $PREFIX/bin/autoscan
-sed -i.bak '1 s|^.*$|#!/usr/bin/env perl|g' $PREFIX/bin/autoupdate
-
 cd $SRC_DIR/ext
 wget -O seqtk.tar.gz https://github.com/lh3/seqtk/archive/d3b53c9.tar.gz
 tar -xvf seqtk.tar.gz -C seqtk --strip-components 1
 cd ..
 
 mkdir -p $PREFIX/bin
+sed -E -i.bak \
+    -e 's/^(CFLAGS|LDFLAGS)=/\1+=/' \
+    -e 's/(\s+)gcc \$\(CFLAGS\)/\1$(CC) $(CPPFLAGS) $(CFLAGS)/' \
+    Makefile.am
```

Comment:
I'm not too familiar with Autotools, so no idea if this works.
If it does, we should open a PR upstream for something like this!